org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_Scannable.java

```diff
@@ -253,6 +254,7 @@ public MonitorRole getMonitorRole() {
 	public MonitorRole setMonitorRole(MonitorRole role) throws ScanningException {
 		if (info==null) update();
 		MonitorRole oldRole = info.getMonitorRole();

```

Comment:
This pattern is used throughout this file, Fixing this function in isolation would make it inconsistent with the rest of the class.